datalad/tests/test_utils.py

```diff
@@ -923,37 +923,47 @@ def test_get_dataset_root(path=None):
         eq_(get_dataset_root(fname), os.curdir)
 
 
-@skip_if_on_windows
+def _p(p: str) -> str:
+    """A helper to code paths as POSIX paths in tests  below. Would prepend fake drive
+       C: to absolute paths on Windows"""
+    if on_windows:
+        pm = p.replace('/', os.sep)
+        if p.startswith('/'):
+            return "C:{pm}"
+        else:
+            return pm
+    return p
+
+
 def test_path_startswith():
```

Comment:
had to fix up accidentally mass renamed `_path_ -> _p_` here ;)  tests should pass and then I will merge -- no code changes, just got tests to pass on exotic Windows. What could go wrong?